src/assets/scripts/client/utilities/validatorUtilities.js

```diff
@@ -0,0 +1,36 @@
+import _isArray from 'lodash/isArray';
+import _isEmpty from 'lodash/isEmpty';
+import _isObject from 'lodash/isObject';
+
+/**
+ * This will return true if it is not an object or it is empty
+ *
+ * @funtion isNotObjectOrIsEmpty
+ * @param input {object}
+ * @return {boolean}
+ */
+export const isObjectEmpty = (value) => {
+    return !_isObject(value) || _isEmpty(value);
```

Comment:
add some tests to make sure the logic is correct here.  we want this to return true only when an object is empty.